src/barometer.cpp

```diff
@@ -3,29 +3,29 @@
 #include <Arduino.h>
 
 namespace obc {
-Result<Unit, Errc> init(Bmp &bmp)
-{
-    Serial.println("Initializing BMP module");
 
-    if (bmp.begin() == 0) { return Err{Errc::Busy}; }
+namespace {
 
-    Serial.println("BMP init success!\n\n");
+constexpr double ground_lvl_pressure = 1013.25;
```

Comment:
czemu to zostało przeniesione do cpp?

Reply:
bo to jest implementation detail, nikt tego nie uzywa poza modułem